### ISSUE TITLE
explorer 

### ISSUE THREAD

**Issue by progamer562** [PhoenixPE/PhoenixPE#61]
how do i stop explorer opening when the pe boots i can't find anything to disable it

**Comment by homes32** [comment]
Assuming you have enabled "Auto Login as Admin" this is a glitch with PECMD spawning extra explorer processes. (see also #49).

I am not the author of PECMD nor do I have the source code for it, so we are left trying to find a workaround.

one method that can help, assuming you don't want to switch back to SYSTEM is to modify PECMD.ini
and add `KILL -explorer explorer.exe` in the _SUB SwitchToAdmin function like so:

```
// Switch User
KILL -explorer explorer.exe
EXEC -hide tsdiscon.exe
```

**Comment by progamer562** [comment]
yes i do let me try the command you posted :)

**Comment by progamer562** [comment]
the command didnt work if i used the system account then i lost audio

**Comment by progamer562** [comment]
_SUB SwitchToAdmin

TEXT Login as Admin...#0xFFFFFF L59 T39 $20

// Reset the flag so it doesn't run again on admin login
REGI #HKLM\Software\Microsoft\Windows NT\CurrentVersion\Winlogon\AutoSwitchToAdmin=0

// we always want to grab the latest copy of the default user hive in case it was modified post-build.
IFEX X:\Users\Default\NTUSER.DAT,FILE X:\Users\Default\NTUSER.DAT
FILE %Windir%\System32\config\Default=>X:\Users\Default\NTUSER.DAT

// Join Workgroup
EXEC -hide -wait SetWG.exe WORKGROUP

// Start the Secondary Logon Service
EXEC -hide -wait sc start seclogon

// Switch User
EXEC -hide tsdiscon.exe
KILL -explorer explorer.exe

_END SwitchToAdmin

**Comment by homes32** [comment]
you have it switched around. the KILL command needs to come before tsdiscon

**Comment by progamer562** [comment]
hey awesome it works next one i want to know and how to do is to add Microsoft edge to my pe image

**Comment by progamer562** [comment]
update the command still didnt work

**Comment by homes32** [comment]
I'm working on making PECMD go away, so hopefully when that happens all these quirks go away as well. ;)

**Comment by homes32** [comment]
Resolved in 1c17ec77c3b58693a3330a88d21a0db8a5b8e76f

**Comment by progamer562** [comment]
Awesome the how do I do it as the release is Aug the 24th
